guide/src/tutorials/draw/drawing-images.md

```diff
@@ -0,0 +1,188 @@
+# Drawing Images
+
+**Tutorial Info**
+
+- Author: Josiah Savary
+- Required Knowledge:
+    - [Getting Started](/getting_started.md)
+    - [Draw a Sketch](./draw-a-sketch.md)
+- Reading Time: 20 minutes
+
+---
+
+In this tutorial, we explore drawing images with nannou. We will cover loading textures from images the app's assets directory and drawing them, and applying basic transformations.
+
+You can also find this file, and other useful examples, in the [examples](https://github.com/nannou-org/nannou/tree/master/examples) directory of the nannou source repository.
+
+To begin with, we will need a nannou project file to work with. Copy the following into new file:
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+use nannou::prelude::*;
+
+struct Model {}
+
+fn main() {
+  nannou::app(model).run()
+}
+
+fn model(_app: &App) -> Model {
+  Model {}
+}
+
+fn view(_app: &App, _model: &Model, _frame: Frame) {
+}
+```
+
+Next, we will also need to bootstrap an `assets` directory with some images at the root of our project. I recommend copying the assets directory from the `nannou` repo itself to get started.
+
+```bash
+git clone https://github.com/nannou-org/nannou.git
+cp -r nannou/assets ./assets
+rm -rf nannou
+```
+
+Once you have a fleshed-out `assets` directory at the root of your project, you'll be ready to return to your `main.rs` file for the following steps.
+
+## Creating a Window
+
+We need somewhere to display the image, so first, let's create a window:
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+# use nannou::prelude::*;
+# struct Model {}
+# fn main() {
+#   nannou::app(model).run()
+# }
+fn model(app: &App) -> Model {
+  // Create a new window!
+  app.new_window().size(512, 512).view(view).build().unwrap();
+  Model {}
+}
+# fn view(_app: &App, _model: &Model, _frame: Frame) {
+# }
+```
+
+If you `cargo run` your app, you'll see an empty green window.
```

Comment:
This is showing me an empty yellow window when I run it... it might be prudent to not specify the exact window color here.